engine/src/main/java/org/hibernate/search/backend/impl/lucene/WorkProcessor.java

```diff
@@ -19,7 +19,7 @@
  * @author Sanne Grinovero (C) 2014 Red Hat Inc.
  * @since 5.0
  */
-interface WorkProcessor {
+public interface WorkProcessor {
```

Comment:
Right; I added a fixup commit. Will `rebase --autosquash` when you're done reviewing, I don't want to mess with the commit hashs until then.